openedx/features/caliper_tracking/transformers/xblock_transformers.py

```diff
@@ -0,0 +1,45 @@
+"""
+Transformers for all the xblock events
+"""
+
+
+def xblock_survey_submitted(current_event, caliper_event):
+    """
+    The server emits an xblock.survey.submitted event each
+    time a user submits responses to a survey.
+
+
+    :param current_event: default event log generated.
+    :param caliper_event: caliper_event log having some basic attributes.
+    :return: updated caliper_event.
+    """
+    caliper_object = {
+        "id": current_event['referer'],
+        "type": 'Assessment',
+        "extensions": current_event['event']
+    }
+
+    caliper_event['extensions']['extra_fields'].update({
+        "ip": current_event['ip'],
```

Comment:
Please use single quotes in all of code. Only use double quotes for docstrings.

Reply:
done in the latest commit.